Options Page Changes to be committed:

NevasPrime opened the pull request:
modified:   nplbam/app/__init__.py
        - Added blueprint for Options page
	new file:   nplbam/app/options.py
        - Added options page which will allow users to change passwords
	new file:   nplbam/app/templates/options.html
        - New Page, uses new_account css for now

NevasPrime committed:
Changes to be committed:
	modified:   nplbam/app/__init__.py
        - Added blueprint for Options page
	new file:   nplbam/app/options.py
        - Added options page which will allow users to change passwords
	new file:   nplbam/app/templates/options.html
        - New Page, uses new_account css for now

ValliereMagic commented on nplbam/app/__init__.py:
```diff
@@ -26,7 +26,7 @@ def create_app():
     from . import (accounts, add_organization, animals, dashboard, edit_animal,
                    edit_organization, file_downloads, main, new_animal,
-                   organizations, view_animal)
+                   organizations, view_animal, options)
```
Import sorting (should be available in right click menu)

NevasPrime committed:
Changes to be committed:
	modified:   nplbam/app/__init__.py
	modified:   nplbam/app/animals.py
	modified:   nplbam/app/options.py
        - Formatting and Sorted Imports
